### PR TITLE
CA-164 - Add S3 server access logging feature to ephemeral spark example

### PR DESCRIPTION
**Pull request by skalish**

### CODE REVIEW

File: examples/ephemeral-spark/local.tfvars [hunk 1]
```diff
@@ -8,4 +8,5 @@ compute_subnet_id     = "subnet-us-east-1b" # Replace me
 data_subnet_ids = [
   "subnet-us-east-1a",
   "subnet-us-east-1b",
-] # Replace me with subnet IDs in different AZs
+]                                        # Replace me with subnet IDs in different AZs
+s3_bucket_logging = "example-log-bucket" # Replace me
```
Comment (souza-dan) [inline examples/ephemeral-spark/local.tfvars:12]: We should be clear that this example will not create the bucket.
```suggestion
s3_bucket_logging = "example-log-bucket" # Replace me with an existing bucket
```

File: examples/ephemeral-spark/variables.tf [hunk 2]
```diff
@@ -75,3 +75,9 @@ variable "create_new_service_role" {
   type        = bool
   description = "Whether to create a new IAM service linked role for ES. This only needs to happen once per account. If false, linked_service_role is required"
 }
+
+variable "s3_bucket_logging" {
+  description = "The name of S3 bucket where to store S3 server access logs."
```
Comment (souza-dan) [inline examples/ephemeral-spark/variables.tf:80]:
```suggestion
  description = "The name of an existing S3 bucket where to store S3 server access logs."
```

**Review by souza-dan — COMMENTED** [review]
We should be clear that this new variable is for a pre-existing bucket, or we could update the example to create this 3rd logs bucket. Either option seems reasonable to me.

**Comment by skalish** [comment]
The pre-existing bucket seems more natural to me based on customer experience (sample size: 1).

**Review by souza-dan — APPROVED** [review]
LGTM

**Pull request merged**